database/base.py

```diff
@@ -43,6 +44,8 @@ class Base(DeclarativeBase):
 ]
 timestamp = Annotated[datetime, mapped_column(server_default=func.now())]
 
+autoincrement_int = Annotated[int, mapped_column(autoincrement=True)]
```

Comment:
Damn, another "one more thing" moment!

Perhaps autoincrement_int should have index=True, because probably we will often/always want to order things in the order that the autoincrement_int specifies. Adding index makes ordering faster.

Then, we could also add a test for the index, in the case that `regulation_number` field exists in a table.

Reply:
Hmm, we could even have a more general `ordering` field name in PlanRegulation. It might be that some other tables want `ordering` in the future, so perhaps `ordering` is a good general name for the field that determines object order in QGIS etc.